PWGCF/GenericFramework/Tasks/flowGenericFramework.cxx

```diff
@@ -100,7 +103,6 @@ struct GenericFramework {
   // Define output
   OutputObj<FlowContainer> fFC{FlowContainer("FlowContainer")};
   OutputObj<FlowContainer> fFC_gen{FlowContainer("FlowContainer_gen")};
-  OutputObj<FlowContainer> fFC_reco{FlowContainer("FlowContainer_reco")};
   OutputObj<GFWWeights> fWeights{GFWWeights("weights")};
   HistogramRegistry registry{"registry"};
   HistogramRegistry QAregistry{"QAregistry", {}, OutputObjHandlingPolicy::QAObject};
```

Comment:
Have you already tested that you get the `QAregistry` on hyperloop?
If I remember correctly the `QAObject` are not obtained when tasks are run on hyperloop but this could have changed

Reply:
The hyperloop test at least return the QAResults.root. Have not gotten the full train output yet.